addons/udes_stock_refactoring/models/refactor_batch_pickings_by_date.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from itertools import groupby
+
+from odoo import models, fields, _
+
+from .refactor import Refactor, get_selection
+
+
+class BatchPickingsByDate(Refactor):
+    """Refactor pickings by scheduled date."""
+
+    @classmethod
+    def name(cls):
+        """Set code name of the refactor action."""
+        return "batch_pickings_by_date"
+
+    @classmethod
+    def description(cls):
+        """Set description of the refactor action."""
+        return "Batch Pickings by Date"
+
+    def do_refactor(self, moves):
+        """Batch pickings by date."""
+        moves._refactor_action_batch_pickings_by(
```

Comment:
Does `_refactor_action_batch_pickings_by` return the moves?

Reply:
No it doesn't. I think that is why there is a ```if new_moves is not None``` check in ```_action_refactor``` in Stock Move.